pinecone_text/dense/openai_encoder.py

```diff
@@ -76,3 +77,41 @@ def _encode(
         if isinstance(texts, str):
             return response.data[0].embedding
         return [result.embedding for result in response.data]
+
+
+class AzureOpenAIEncoder(OpenAIEncoder):
```

Comment:
Can you please add system and unit tests same as we have for OpenAI non Azure embeddings?